core/include/detray/propagator/rk_stepper.ipp

```diff
@@ -27,20 +27,22 @@ void detray::rk_stepper<magnetic_field_t, transform3_t, constraint_t, policy_t,
     auto dir = track.dir();
 
     // Update the track parameters according to the equations of motion
-    pos = pos + h * (dir + h_6 * (sd.k1 + sd.k2 + sd.k3));
+    // Reference: Eq (82) of https://doi.org/10.1016/0029-554X(81)90063-X
+    pos = pos + h * (sd.t[0u] + h_6 * (sd.dtds[0] + sd.dtds[1] + sd.dtds[2]));
```

Comment:
E.g. in the equation you cite here, the stages are named ```K_n```...

Reply:
K == dtds as mentioned above